### DISCUSSION TITLE
Replacing \n with <br>!

### DISCUSSION THREAD

**Discussion by coltmay** [nss-day-cohort-47/C47-Discussions#64] [Look what I learned!]
If you're like me, and you want to keep your breaks consistent between input forms and output displays.  Then preserving line breaks is pretty damn essential.

After some tinkering and googling, I found this:

```javascript
// for any string
string = string.replace(/\n/g, "<br>")
```
This will find any new lines in your text `\n` and convert them into HTML friendly `<br>`.  I believe the `g` in this case is a _global modifier_, so it will replace all occurrences of `\n`.

**Comment by BryanNilsen** [comment]
This is great!! Welcome to regular expressions! 
you are correct that the `g` "flag" refers to a "global" search for the matching characters

In case anyone wants to explore... here's the link to the [Regular Expressions info on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)
